docs/ja/front/introduction.md

```diff
@@ -1,11 +1,11 @@
 # はじめに
 
-企業や組織（総称して「組織」）は、オープンソースやその他のソフトウェアパッケージを広く使用し、再利用しています。ソフトウェアの正確な識別は、多くのサプライチェーンプロセスにとって重要です。脆弱性の修復は、システムで使用されているソフトウェアのバージョンの詳細を知ることから始まります。関連するライセンスのコンプライアンスには、各組織が独自に行う一連の分析活動とデューデリジェンスが必要であり、これには、ソフトウェアの手動スキャンや自動スキャン、関連するライセンスの識別、その後の手動検証などが含まれます。
+企業や組織（総称して「組織」）は、オープンソースやその他のソフトウェアパッケージを広く使用し、再利用しています。ソフトウェアを正確に識別することは、多くのサプライチェーンプロセスにとって重要です。脆弱性の修復は、システムで使用されているソフトウェアのバージョンの詳細を知ることから始まります。関連するライセンスのコンプライアンスには、各組織が独自に行う一連の分析活動とデューデリジェンスが必要であり、これには、ソフトウェアの手動スキャンや自動スキャン、関連するライセンスの識別、その後の手動検証などが含まれます。
 
-世界中のソフトウェア開発チームが同じオープンソースパッケージを使用していますが、分析に関する共同作業を促進したり、分析結果を共有したりするためのインフラはほとんど存在しません。その結果、多くのグループが同じ作業を行っており、重複した作業や冗長な情報が発生しています。このドキュメントでは、Linux Foundation SPDX グループと OMG/CISQ Tool-to-Tool ワークグループの統合作業である SPDX ワークグループが、ソフトウェアパッケージと関連コンテンツに関する情報を収集し、共通のフォーマットで共有することで、時間の節約とデータの正確性の向上を目的としたデータ交換フォーマットを作成しました。
+世界中のソフトウェア開発チームが同じオープンソースパッケージを使用していますが、分析に関する共同作業を促進したり、分析結果を共有したりするためのインフラはほとんど存在しません。その結果、多くのグループが同じ作業を行っており、重複した作業や冗長な情報が発生しています。このドキュメントは、Linux Foundation SPDX グループと OMG/CISQ Tool-to-Tool ワークグループの統合的な活動である SPDX ワークグループが作成した、ソフトウェアパッケージと関連コンテンツに関する情報を収集し、共通のフォーマットで共有することで、時間の節約とデータの正確性の向上を目的としたデータ交換フォーマットについて説明しています。
 
-この2つのグループの統合された活動は、2021年の最初の数週間、概ね前進しましたが、どちらか一方が議論していなかったり、異なる意見を持っていたりする問題を、より大きなグループが解決する間、時折停滞しました。最終的に、SPDX 3.0が象徴するシフトに備え、SPDX 3.0のコンセプトや機能の一部をコミュニティに提供するアップデートを加えたSPDX 2.3を2022年8月にリリースした後、SPDX 3.0の最初のリリース候補が2023年5月にリリースされました。標準化団体であると同時にオープンソース開発者のコミュニティでもあるSPDXコミュニティにおいて、リリース候補版は、新旧を問わずSPDXの実装者が作業をレビューし、不明確な部分や実装に大きな負担となる部分がないかを判断する機会を提供します。
+この2つのグループの統合的な活動は、2021年の最初の数週間、概ね前進しましたが、どちらか一方が議論していなかったり、異なる意見を持っていたりする問題を、より大きなグループが解決する間、時折停滞しました。最終的に、SPDX 3.0での方向転換に備え、SPDX 3.0のコンセプトや機能の一部をコミュニティに提供するアップデートを加えたSPDX 2.3を2022年8月にリリースした後、SPDX 3.0の最初のリリース候補版が2023年5月に公開されました。標準化団体であると同時にオープンソース開発者のコミュニティでもあるSPDXコミュニティにおいて、RC版(リリース候補版)は、新旧を問わずSPDXの実装者が成果物をレビューし、不明確な部分や実装に大きな負担となる部分がないかを判断する機会を提供しました。
 
-最初のリリース候補からのコメントや変更要求に基づいて、モデルのいくつかの領域が修正され、手直しされた結果、2024年2月にSPDX 3.0のリリース候補2がリリースされました。このリリース候補は、SPDX 3.0仕様の最終バージョンに先立ち、ツール作成者やSPDXで作業するためのサポートライブラリを保守する人たちが、プロジェクトの改訂を開始する時間を与えるものでした。SPDX 2.3からSPDX 3.0に移行する中で、SPDXモデルには劇的な変化がありました。SPDXの名称がSoftware Package Data ExchangeからSystem Package Data Exchangeに変更され、部品表で伝達できる項目がソフトウェア、セキュリティ、ライセンスから、データセット、AIモデル、ビルド情報などの多くの追加要素に拡大されました。
+最初のRC版に対するコメントや変更要求に基づいて、モデルのいくつかの領域が修正され、再検討された結果、2024年2月にSPDX 3.0のRC2版が公開されました。このRC2版は、ツール作成者やSPDXで作業するためのサポートライブラリをメンテナンスする人たちに、SPDX 3.0仕様の最終バージョンに先立ち、プロジェクトの改訂を開始する時間を与えるものでした。コミュニティ内部の活動や議論、SPDXグループとOMG/CISQ Tool-to-Tool (3T-SBOM)ワークグループとの統合の議論に関与していない人たちにとっては、この3年間でのSPDX 2.3からSPDX 3.0へのSPDXモデルの移行は、Software Package Data ExchangeからSystem Package Data ExchangeへのSPDXの名称変更、部品表で伝達できる項目の拡大(ソフトウェア、セキュリティ、ライセンスから、データセット、AIモデル、ビルド情報などの多くの要素が追加された)など、劇的な変化となりました。
```

Comment:
> it can now convey in a bill of materials from software, security, and licensing to many additional aspects like data sets, AI models, and build information.

from ... to... のニュアンス出すため

「多くの要素**まで**が追加された」ってするのはどうでしょうか?

参考までに該当箇所を含む一文の別の機械翻訳(一部修正)したものも貼っておきます

「過去3年間、3T-SBOMとSPDX 3.0の合同ワーキンググループの内部の動きや議論、討論に注目していなかった人にとって、SPDXモデルがSPDX 2.3からSPDX 3.0へと移行する中で劇的な変化があり、SPDXの名称が「Software Package Data Exchange（ソフトウェアパッケージデータ交換）」から「System Package Data Exchange（システムパッケージデータ交換）」へと変更され、部品表において伝えることができる項目の範囲が、ソフトウェア、セキュリティ、ライセンスから、データセット、AIモデル、ビルド情報といった多くの追加の側面にまで拡大されたことは驚きかもしれません。」

Reply:
今回の「OSSJにとにかく間に合わせるぞ」版については、私が修正するとかはしません（せいぜい、一度声に出して読んでみるくらい？）。なので、PRされた方の余裕次第かな。と思います。

ここに書いてあれば、宿題、ということにできるかと思いますし。